peps/pep-0752.rst

```diff
@@ -9,6 +9,7 @@ Type: Standards Track
 Topic: Packaging
 Created: 13-Aug-2024
 Post-History: `18-Aug-2024 <https://discuss.python.org/t/61227>`__,
+              `07-Sep-2024 <https://discuss.python.org/t/63191>`__,
```

Comment:
That's the PEP 755 thread, here's the new 752 one:
```suggestion
              `07-Sep-2024 <https://discuss.python.org/t/63192>`__,
```

And let's update `Discussions-To` to point to this new one as well.

Reply:
Good catch, thank you!